clippy_config/src/conf.rs

```diff
@@ -549,6 +549,10 @@ define_Conf! {
     /// The maximum size of the `Err`-variant in a `Result` returned from a function
     #[lints(result_large_err)]
     large_error_threshold: u64 = 128,
+    /// Whether collapsible `if` chains are linted if they contain comments inside the parts
+    /// that would be collapsed.
+    #[lints(collapsible_if)]
+    lint_commented_code: bool = true,
```

Comment:
```suggestion
    lint_commented_code: bool = false,
```

If there is a comment between the 2 if stmts, it is a strong indication that the separation is intentional and for readability.

Reply:
Done